Helm/opensearch/values.yaml

```diff
@@ -311,7 +330,39 @@ fullnameOverride: ""
 
 masterTerminationFix: false
 
-lifecycle: {}
+probes:
+  startup:
+    enabled: true
+    interval: 5
+    delay: 15
+  # Enabled by default, utilizes tcpSocket to check the transport port of the pod
+  liveness:
+    enabled: true
+    interval: 10
+  # ensure the username and password used for READINESS_USERNAME and READINESS_PASSWORD has permissions to access /_cat/health otherwise the pod will never start
+  readiness:
+    enabled: false
+    interval: 10
+    delay: 30
+    # Default utilizes READINESS_USERNAME and READINESS_PASSWORD to query _cat/health for status
+    default: true
+    username: admin
+    password: admin
```

Comment:
See previous comments. This is a good way forward but it could be cleaner and shouldn't really be optional.